charts/hawkbit/templates/pvc.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.fileStorage.enabled -}}
```

Comment:
how about adding a license header?

Reply:
None of the files do have a license header in the hawkbit package. I agree it would be an idea to add them. But I would then do it in another PR, outside of the scope of this one.